src/lineup/internal/export.ts

```diff
@@ -113,7 +114,7 @@ interface IExportData {
 
 function customizeDialog(provider: LocalDataProvider): Promise<IExportData> {
   return lazyDialogModule().then((dialogs) => {
-    const dialog = new dialogs.FormDialog('Export Data &hellip;', '<i class="fa fa-download"></i> Export');
+    const dialog = new dialogs.FormDialog(`${i18n.t('tdp:core.lineup.export.exportData')} &hellip;`, `<i class="fa fa-download"></i>${i18n.t('tdp:core.lineup.export.export')}`);
```

Comment:
Please add the `&hellip;` to the language string as some customizations might want to change this...